terraform-state-backend.template

```diff
@@ -188,6 +188,8 @@ Resources:
         KeyType: HASH
       PointInTimeRecoverySpecification:
         PointInTimeRecoveryEnabled: true
+      ContributorInsightsSpecification:
```

Comment:
Does the title of this pull request ("consumed read capacity units") relate to contributor insights?